src/makeInfinityAdder.js

```diff
@@ -4,7 +4,22 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+    let sum = 0;
+
+  function adder(num) {
+    if (num === undefined) {
+      const newSum = sum;
```

Comment:
Storing the sum in a new variable `newSum` is unnecessary. You can simply return `sum` and then reset it to `0` on the next line.